google/cloud/storage/client.h

```diff
@@ -908,7 +959,34 @@ class Client {
                destination_object_metadata, std::forward<Options>(options)...)
         .Result();
   }
+  //@}
 
+  //@{
+  /**
+   * @name Bucket Access Control List operations.
+   *
+   * You can control who has access to your GCS buckets and objects as
+   * well as what level of access they have.
+   *
+   * @note In most cases, you should use IAM permissions instead of ACLs to
+   * control access to buckets.
+   *
+   * @note All buckets are owned by the project owners group. Project owners
+   *     are granted `OWNER` permissions to all buckets inside their project.
+   *     Bucket and object ownership cannot be changed by modifying ACLs. If you
+   *     apply a new ACL to a bucket, be sure that the bucket owner remains
+   *     unchanged in the new ACL.
```

Comment:
I'm confused as to why this last sentence is present.  If users can't change the owner via ACL operations, whatever they do in the ACL update operation will have no effect on the original OWNER-role entry.  Maybe a more accurate wording might be something like what we have in the gsutil docs: "You cannot remove the project owners group from a bucket ACL. Attempts to do so will appear to succeed, but the service will add the project owners group into the new  ACL before applying it."

Reply:
I agree.  Removed the old wording and used yours.  Note that I took that wording more or less verbatim from here:

https://cloud.google.com/storage/docs/access-control/lists#bestpractices